makefiles/deprecated_modules.inc.mk

```diff
@@ -1,3 +1,4 @@
 # Add deprecated modules here
 # Keep this list ALPHABETICALLY SORTED!!!!111elven
 DEPRECATED_MODULES += event_thread_lowest
+DEPRECATED_MODULES += gnrc_netdev_default
```

Comment:
Does it make sense to add an EOL release here? For `event_thread_lowest` this did not seem to be strictly necessary, as it mostly provides an alias for backwards-compatibility, but I think for `gnrc_netdev_default` we _do_ want to remove it at some point.

Reply:
Problem is: it won't show up in https://doc.riot-os.org/deprecated.html, where [the wiki](https://github.com/RIOT-OS/RIOT/wiki/Managing-a-Release#2-preparation) recommends to look for deprecated features.